AEPMessaging/Sources/Event+Messaging.swift

```diff
@@ -226,6 +227,22 @@ extension Event {
     var offerPlacementId: String? {
         placement?[MessagingConstants.Event.Data.Key.Optimize.ID] as? String
     }
+    
+    var offerDecisionScope: String? {
+        guard let payload = payload, !payload.isEmpty else {
+            return nil
+        }
+        
+        guard let b64EncodedScope = payload.first?[MessagingConstants.Event.Data.Key.Optimize.SCOPE] as? String else {
+            return nil
+        }
+        
+        guard let scopeData = Data(base64Encoded: b64EncodedScope), let scopeDictionary = try? JSONSerialization.jsonObject(with: scopeData, options: .mutableContainers) as? [String: Any] else {
```

Comment:
Any specific requirement to keep the JSONSerialization.ReadingOptions as `mutableContainers ` ?